# RNA-seq-with-snakemake

In this video, I‚Äôll walk you through a fully automated, one-click RNA-seq pipeline using Snakemake. Starting from raw FASTQ files, you‚Äôll see how to generate a final table of differentially expressed genes‚Äîall in a single command.

Rather than executing 19+ steps manually, we‚Äôll use workflow automation to streamline the entire process. This is not a beginner's command-by-command tutorial‚Äîthat‚Äôs covered in another video. Instead, this is for those who want to level up their bioinformatics skills by adopting reproducible, scalable, and modular workflows.

Whether you're a student or a seasoned researcher, by the end of this video, you'll understand:

How Snakemake traces back the required rules using wildcards
How to interpret a pipeline DAG (Directed Acyclic Graph)
How dry runs reveal the logic of workflow propagation
And how to combine Snakemake with Docker to ensure reproducibility

üîó Code & metadata available in the description below.


‚è±Ô∏è Chapters
00:00 Intro: One-Click RNA-seq with Snakemake
00:57 Visualizing the Workflow with DAG
02:28 Dry Run Setup (Understanding Dependencies)
03:12 How Snakemake Backtracks Using Wildcards
07:03 Docker Integration for Reproducibility
07:35 Final DEGs Table
07:56 Outro
